pages/services/kubernetes/2.0.1-1.12.2/release-notes/index.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: layout.pug
+navigationTitle: Release Notes
+title: Release Notes
+menuWeight: 1
+excerpt: Release notes for DC/OS Kubernetes version 2.0.1-1.12.2
+---
+
+<!-- This source repo for this topic is https://github.com/mesosphere/dcos-kubernetes-cluster -->
+
+# Version 2.0.1-1.12.2
+
+## Breaking Changes
+
+* DC/OS Kubernetes `2.0.1-1.12.2` requires DC/OS 1.12.
+* DC/OS Kubernetes `2.0.1-1.12.2` introduces breaking changes to the way the package works and is deployed.
```

Comment:
Let's remove or clarify that these breaking changes only apply to 1.x > 2.x. Not 2.0.0 > 2.0.1